ansible/roles/stack-sunbird/tasks/content-service.yml

```diff
@@ -16,6 +20,15 @@
   become: yes
   shell: "docker config create content-service.conf /home/deployer/config/content-service.conf"
 
+- name: Remove old logback.xml docker config
+  become: yes
+  shell: "docker rm /home/sunbird/content-service-1.0-SNAPSHOT/config/logback.xml"
```

Comment:
@PradyumnaNagendra What are we doing here? docker rm is used to remove containers that are in stopped state and not config files.